forge/src/main/java/org/openjdk/skara/forge/Forge.java

```diff
@@ -59,7 +59,11 @@ static Forge from(String name, URI uri, Credential credential) {
     }
 
     static Forge from(String name, URI uri) {
-        return from(name, uri, null);
+        return from(name, uri, (Credential) null);
```

Comment:
Not entirely sure I like this. Maybe "from" has been overloaded too much?

Reply:
Or you could return `from(name, uri, null, null)`, could you not? Anything but casting null...